apis/sauce.json

```diff
@@ -0,0 +1,2325 @@
+{
```

Comment:
I also got the `swagger.json` for RDC, you can get it from [here](https://app.testobject.com/api/rest/swagger.json)

Reply:
awesome, will add it